server/budgettracker.business/Api/Contracts/BudgetApi/UpdateBudget/UpdateBudgetRequestContract.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using Newtonsoft.Json;
+
+namespace budgettracker.business.Api.Contracts.BudgetApi.UpdateBudget
+{
+    public class UpdateBudgetRequestContract : IApiContract {
+
+        [JsonProperty("id")]
+        public Guid Id { get; set; }
+
+        [JsonProperty("name")]
+        public string Name { get; set; }
+
+        [JsonProperty("set-amount")]
+        public decimal SetAmount { get; set; }
+
+        /// <summary>
+        /// <p> The duration of the budget cycle in days.</p>
+        /// </summary>    
+        [JsonProperty("duration")]
+        public int Duration { get; set; }
```

Comment:
FYI for whoever merges first, #53 makes Duration an object with other fields. Whoever merges last will have to update their Code to reflect that. Don't worry about it if you merge first but If I merge first, we'll have to communicate about how to change it.